R/cookies.R

```diff
@@ -203,190 +203,236 @@ cookie_banner_server <- function(
   })
 }
 
-#' cookies_panel_ui
+#' init_cookies
 #'
 #' @description
-#' Create the standard DfE R-Shiny cookies dashboard panel in the ui. The server
-#' side functionality is provided by cookies_panel_server()
+#' init_cookies() creates a local copy of the JavaScript file
+#' required for cookies to work.
+#' It checks whether there is already a www/ folder and if not, it creates one
```

Comment:
Can you add full-stops at the end of these lines. When it renders in a help page, it takes out the carriage returns and just flows on, which makes it awkward to read if there's no full-stops at the end of sentences.